scout/server/blueprints/cases/views.py

```diff
@@ -748,6 +759,43 @@ def rerun(institute_id, case_name):
     return redirect(request.referrer)
 
 
+@cases_bp.route("/<institute_id>/<case_name>/reanalysis", methods=["POST"])
+def reanalysis(institute_id, case_name):
+    """Toggle a rerun by making a call to RERUNNER service."""
+    LOG.info("Building query for reanalysis")
+    edited_metadata = json.loads(request.form["sample_metadata"])
+
+    # define the data to be passed
+    payload = {"case_id": case_name, "sample_ids": [s["sample_id"] for s in edited_metadata]}
+
+    cnf = rerunner.connection_settings
+    url = f"http://{cnf.get('host')}/v1.0/rerun"
+    auth = HTTPBasicAuth(current_user.email, cnf.get("api_key"))
+    LOG.info(f"Sending request -- {url}; params={payload}")
+    try:
+        resp = requests.post(
+            url,
+            params=payload,
+            json=edited_metadata,
+            timeout=rerunner.timeout,
+            headers={"Content-Type": "application/json"},
+            auth=auth,
+        )
+        if resp.status_code == requests.codes.ok:
+            LOG.info(f"Reanalysis was successfully started; case: {case_name}")
+            flash(f"Reanalysis was successfully started; case: {case_name}", "info")
```

Comment:
Apart from this flash message that you see only once, there is no saved information that the rerun is pending at the case level. What about triggering an event that will update the case "rerun_requested" and set it to True, and will also display which user ordered a rerun? Consider calling [request rerun](https://github.com/Clinical-Genomics/scout/blob/5485aa5f9731bffa02fdfebbcd42783d50350071/scout/adapter/mongo/case_events.py#L338) from here.